code/src/components/Important.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+import { tasks } from "reducers/tasks";
+
+import styled from "styled-components";
+
+export const Important = ({ ifImportant, id }) => {
+  const dispatch = useDispatch();
+
+  const handleClick = () => {
+    dispatch(tasks.actions.toggleImportant(id));
+  };
+  return (
+    <Button type="button" onClick={handleClick}>
+      {ifImportant === true && (
+        <ion-icon name="star" style={{ color: "orange" }}></ion-icon>
+      )}
+      {ifImportant === false && (
+        <ion-icon name="star-outline" style={{ color: "orange" }}></ion-icon>
+      )}

```

Comment:
you could shorten this like:
`<ion-icon name={ifImportant ? "star" : "star-outline"} style={{ color: "orange" }}></ion-icon>`
since the name is the only thing that's different

Reply:
True! Good point! : ) will fix that